modules/commands/list.json

```diff
@@ -500,6 +518,56 @@
       }
     ]
   },
+  {
+    "name": "ssl",
+    "aliases": [
+      "usessl"
+    ],
+    "title": "The useSSL error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",
+    "fields": [
+      {
+        "key": "Read me!",
+        "value": "https://github.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors"
+      },
+      {
+        "key": "What is SSL?",
+        "value": "https://www.cloudflare.com/learning/ssl/what-is-ssl/"
+      }
+    ]
+  },
+  {
+    "name": "timeout",
+    "aliases": [
+      "connectiontimeout"
+    ],
+    "title": "The connectionTimeout error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",

```

Comment:
How is this any different from the command which links to the storage errors wiki page?

Don't see why we need to double up.

Reply:
This is mainly just for convenience, in that we can link straight to the really common error and that's about it 🤷‍♂️ 

I'll remove it when I can if you don't tell me not to

Reply:
Also, why does it say `The connectionTimeout error!` (Why even connectionTimeout and not just Connection Timeout?) when it is about the SSL error?
Did you just forget to update the links and fields here? because it looks like it.

Reply:
Yes, I did, it's a leftover, sorry.

Reply:
removed